spec/locales-currencies-tz.html

```diff
@@ -9,21 +9,20 @@ <h1>Identification of Locales, Currencies, Time Zones, and Measurement Units</h1
     <h1>Case Sensitivity and Case Mapping</h1>
 
     <p>
-      The String values used to identify locales, currencies, scripts, and time zones are interpreted in an ASCII-case-insensitive manner, treating the code units 0x0041 through 0x005A (corresponding to Unicode characters LATIN CAPITAL LETTER A through LATIN CAPITAL LETTER Z) as equivalent to the corresponding code units 0x0061 through 0x007A (corresponding to Unicode characters LATIN SMALL LETTER A through LATIN SMALL LETTER Z), inclusive. No other case folding equivalences are applied.
+      The String values used to identify locales, currencies, scripts, and time zones are interpreted in an ASCII-case-insensitive manner, treating the code units 0x0041 through 0x005A (corresponding to Unicode characters LATIN CAPITAL LETTER A through LATIN CAPITAL LETTER Z) as equivalent to the corresponding code units 0x0061 through 0x007A (corresponding to Unicode characters LATIN SMALL LETTER A through LATIN SMALL LETTER Z), both inclusive. No other case folding equivalences are applied.
     </p>
+    <emu-note>
+      For example, *"ß"* (U+00DF) must not match or be mapped to *"SS"* (U+0053, U+0053). *"ı"* (U+0131) must not match or be mapped to *"I"* (U+0049).
+    </emu-note>
     <p>
-      Mapping to upper case takes a String value and produces a new String value in which each lower case ASCII letter code unit (0x0061 through 0x007A, inclusive) is replaced with the corresponding upper case ASCII letter code unit (0x0041 through 0x005A, inclusive) and all other code units are preserved.
+      The <dfn>ASCII-uppercase</dfn> of a String value _S_ is the String value derived from _S_ by replacing each occurrence of an ASCII lowercase letter code unit (0x0061 through 0x007A, inclusive) with the corresponding ASCII uppercase letter code unit (0x0041 through 0x005A, inclusive) while preserving all other code units.
```

Comment:
are these things that could live in 262 and be used by toUpperCase?

Alternatively, why not use the phrasing in 262:
> the result of toUppercase(sText), according to the Unicode Default Case Conversion algorithm.

Reply:
I expect these to migrate into 262 as part of Temporal (which needs ASCII-case-insensitive time zone name comparisons), but they won't use Unicode to{Upper,Lower}case or be used in the algorithms of ECMAScript `to{Upper,Lower}Case` because their transformations are explicitly restricted to the ASCII range (e.g., unlike toUppercase and `toUpperCase`, ASCII-uppercase must not transform "ö").

Reply:
Thanks for clarifying.